### PR TITLE
Use expression body for accessors

### PR DESCRIPTION
**Pull request by Aergwyn**

### CODE REVIEW

File: osu.Framework/Graphics/Containers/TextFlowContainer.cs [hunk 1]
```diff
@@ -40,10 +41,11 @@ public float FirstLineIndent
         /// </summary>
         public float ContentIndent
         {
-            get { return contentIndent; }
+            get => contentIndent;
             set
             {
-                if (value == contentIndent) return;
+                if (value == contentIndent)
```
Comment (peppy) [inline osu.Framework/Graphics/Containers/TextFlowContainer.cs]: any reason for the rearranging of these?

Comment (Aergwyn) [reply osu.Framework/Graphics/Containers/TextFlowContainer.cs]: Most likely autoformatting.

Comment (Aergwyn) [reply osu.Framework/Graphics/Containers/TextFlowContainer.cs]: Definitely autoformatting on the properties when it was changing the code to the expression.

**Comment by peppy** [comment]
I think we still want to keep these cases single-line.

**Comment by Aergwyn** [comment]
I can do that. Is there some style guide in general? Anything I can apply to my options. So I can prevent these automatic changes in the future without manually removing them again.

File: osu.Framework/Configuration/Bindable.cs [hunk 2]
```diff
@@ -64,10 +65,11 @@ public bool Disabled
...
-            get { return value; }
+            get => value;
             set
             {
-                if (EqualityComparer<T>.Default.Equals(this.value, value)) return;
+                if (EqualityComparer<T>.Default.Equals(this.value, value))
+                    return;
 
                 if (Disabled)
                     throw new InvalidOperationException($"Can not set value to \"{value.ToString()}\" as bindable is disabled.");
```
Comment (Aergwyn) [inline osu.Framework/Configuration/Bindable.cs]: Example for the one-liner not being consistent would be this.

**Pull request merged**